### ISSUE TITLE
Add "how to create a page" contributor guide

### ISSUE THREAD

**Issue by DannyS03** [libp2p/docs#252]
**Add the ipfs-docs contributor guide being worked on when ready.**

## Done Criteria
- [ ] Adapt https://github.com/ipfs/ipfs-docs/pull/1294 and libp2p-ize it.

## Why Important
- Great resources for contributors especially first-time contributors.
- Encourages contributions

**Comment by DannyS03** [comment]
@TMoMoreau do you want to take this on and revise the existing doc to match the libp2p docs site/repo?

**Comment by TMoMoreau** [comment]
Absolutely, should be fairly straight forward. Do you have any ideas where this would live already? As in, does the libp2p docs repo have a similar `CONTRIBUTING.md` file already, or should this live in the actual docs somewhere?

**Comment by DannyS03** [comment]
We have a Contributor section that this could go under: https://docs.libp2p.io/concepts/contribute/community/
